src/class_resolver/api.py

```diff
@@ -33,11 +33,13 @@ def __init__(
         :param classes: A list of classes
         :param base: The base class
         :param default: The default class
-        :param suffix: The optional shared suffix of all classes
+        :param suffix: The optional shared suffix of all classes. If None, use the base class' name for it.
```

Comment:
Should mention that if you want alternative behavior (like no suffix) to give an empty string.

Reply:
09e0007